rush/rush/src/actions/InstallAction.ts

```diff
@@ -104,12 +104,12 @@ export default class InstallAction extends CommandLineAction {
       actionVerb: 'install',
```

Comment:
There's an interesting bug with publish where it consumes and applies the version bump even if shouldPublish is false, although it doesn't actually publish the package.